### ISSUE TITLE
[BITV]: missing labels on textaria inside of translate modal dialog

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/text#5221]
![Screenshot from 2024-01-10 10-19-03](https://github.com/nextcloud/text/assets/6078378/86fb61cc-0342-4cb0-a69f-64fbfe9abd9f)

Please adapt `label` for the textaria, `NcTextArea` component

Please check other places on existing `label`'s.

**Comment by luka-nextcloud** [comment]
@JuliaKirschenheuter @juliushaertl Got it, what could be the texts for the labels of textarea elements?
"Input" and "Output"?

**Comment by juliushaertl** [comment]
Maybe "Text to translate from" "Translated text result" to be a bit less technical

**Comment by szaimen** [comment]
I cannot test this but the backport seems to be merged
